INaturalistIOS/ObservationViewCell.h

```diff
@@ -0,0 +1,22 @@
+//
+//  ObservationViewCell.h
+//  iNaturalist
+//
+//  Created by Eldad Ohana on 7/2/15.
+//  Copyright (c) 2015 iNaturalist. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+static const int ObservationCellImageTag = 5;
```

Comment:
Moving forward, I'm hoping to get away from using UIView tags and towards using instance variables. 
Tags are just too error prone with the casting and the static variables . Would you mind re-doing this using instance variables, IBOutlet and Interface Builder?